rules/exposure-to-internet-via-istio-ingress/rule.metadata.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "exposure-to-internet-via-istio-ingress",
+  "attributes": {
+    "useFromKubescapeVersion": "v3.0.9"
+  },
+  "ruleLanguage": "Rego",
+  "match": [
+    {
+      "apiGroups": [
+        ""
+      ],
+      "apiVersions": [
+        "v1"
+      ],
+      "resources": [
+        "Pod",
+        "Service"
+      ]
+    },
+    {
+      "apiGroups": [
+        "apps"
+      ],
+      "apiVersions": [
+        "v1"
+      ],
+      "resources": [
+          "Deployment",
+          "ReplicaSet",
+          "DaemonSet",
+          "StatefulSet"
+      ]
+    },
+    {
+      "apiGroups": [
+        "batch"
+      ],
+      "apiVersions": [
+        "*"
+      ],
+      "resources": [
+          "Job",
+          "CronJob"
+      ]
+    },
+    {
+      "apiGroups": [
+        "networking.istio.io"
+      ],
+      "apiVersions": [
+        "v1"
+      ],
+      "resources": [
+        "VirtualService",
+        "Gateways"
```

Comment:
as you said this is very strange to have plural here

Reply:
Indeed, this is how it worked 😞